src/client/containers/ConfigurationCreateContainer/hooks/useConfigurationCreate.ts

```diff
@@ -58,62 +60,64 @@ export const useConfigurationCreate = ({
                                     },
-                                    workspaceId,
-                                    events
-                                },
-                                microsoftTeams: {
-                                    channel: {
-                                        id: channelId,
-                                        name: channelName
-                                    },
-                                    tenantId,
-                                    incomingWebhookUrl: webhookUrl
-                                }
-                            });
+                                    microsoftTeams: {
+                                        channel: {
+                                            id: channelId,
+                                            name: channelName
+                                        },
+                                        tenantId,
+                                        incomingWebhookUrl: webhookUrl
+                                    }
+                                });
 
-                        microsoftTeams.settings.setSettings({
-                            entityId: configurationId,
-                            configName: resource.name,
-                            contentUrl: decodeURI(`${window.location.origin}${url.getConfigurationUpdateUrl({
-                                id: configurationId,
-                                resourceName: resource.name,
-                                resourceType: resource.type,
-                                channel: "{channelName}",
-                                theme: "{theme}"
-                            })}`)
-                        } as microsoftTeams.settings.Settings);
+                            microsoftTeams.settings.setSettings({
+                                entityId: configurationId,
+                                configName: resource.name,
+                                contentUrl: decodeURI(`${window.location.origin}${url.getHomeUrl({
+                                    id: configurationId,
+                                    resourceName: resource.name,
+                                    resourceType: resource.type,
+                                    channel: "{channelName}",
+                                    theme: "{theme}"
+                                })}`)
+                            } as microsoftTeams.settings.Settings);
 
-                        saveEvent.notifySuccess();
-                    } catch (error) {
-                        saveEvent.notifyFailure(error?.message ?? error);
-                    }
+                            saveEvent.notifySuccess();
+                        } catch (error) {
+                            saveEvent.notifyFailure(error?.message ?? error);
+                        }
+                    });
                 });
             });
-        });
+        }
     }, [resource, events]);
```

Comment:
I think, we should add `isInitialized` to the list, right?

```suggestion
    }, [resource, events, isInitialized]);
```